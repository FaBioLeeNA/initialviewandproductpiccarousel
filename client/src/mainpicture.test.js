require('babel-polyfill');
import React from 'react';
import Enzyme, {shallow} from 'enzyme';
import Mainpic from './components/zoomableMainpic.jsx';
import Styled from './styledComponents/styles.jsx';


describe('Testing for Main Pic to be rendering properly', () => {
  it('Should render the app initially', () => {
    const mainpic = shallow(<Mainpic />);
    expect(mainpic).toMatchSnapshot();
  });
});
