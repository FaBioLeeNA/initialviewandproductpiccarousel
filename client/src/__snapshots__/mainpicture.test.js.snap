// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing Main Pic Should render the app initially 1`] = `ShallowWrapper {}`;

exports[`Testing for Main Pic to be rendering properly Should render the app initially 1`] = `ShallowWrapper {}`;
